cursorless-talon/src/cheatsheet_html/cheat_sheet.py

```diff
@@ -13,15 +12,20 @@
 
 mod = Module()
 
-cheatsheet_out_dir = Path(tempfile.mkdtemp())
 instructions_url = "https://www.cursorless.org/docs/"
 
 
 @mod.action_class
 class Actions:
     def cursorless_cheat_sheet_show_html():
-        """Show new cursorless html cheat sheet"""
-        cheatsheet_out_path = cheatsheet_out_dir / "cheatsheet.html"
+        """Show cursorless html cheat sheet"""
+
+        # NB: We use the user's home directory instead of temp to make sure that
+        # Linux snaps work
+        cheatsheet_out_dir = Path.home() / ".cursorless" / "cheatsheet"
```

Comment:
For follow-up: it might be weird for Windows users to see a dotfile folder in their home, but I don't have any suggestions on where to put it instead other than somewhere in %APPDATA%. Maybe under the Talon home, which is also in AppData?

Reply:
If we want it hidden from the user we can just still use the temp directory in windows. The argument that the user might want access to the files would suggest the user directory. Personally I would prefer the temp directory but I could see the argument for user accessibility.